### ISSUE TITLE
subprocesses.Popen and LIME

### ISSUE THREAD

**Issue by lnieblar** [kpolsen/GMC#2]
In `gmc.py` for `runLIMEmodels(d_models)`,

When I use the `sub.Popen` method, the terminal executes lime, outputs the errors, but doesn't show the progress bars. It outputs the first FITS file, but it doesn't do anything else after that.

I think this is because you need to press a button in LIME when the fits file is finished, and since we're not on that screen lime doesn't want to tell python it is done.

Any ideas? The quick fix for now is just using `sub.call()` method, but I would like to explore subprocesses to see if we can compile multiple model files at a time to reduce runtime.

**Comment by kpolsen** [comment]
Yes I agree that the problem is probably LIME waiting for input. In curses.c in the src folder you can locate the following lines:

void
goodnight(int initime, char filename[80]){
    int runtime=time(0)-initime;
    move(14,4); printw("Output written to %s", filename);
    move(22,0); printw("*** Program ended succesfully               ");
    move(22,58); printw("runtime: %3dh %2dm %2ds", runtime/3600, runtime/60%60, runtime%60);
    move(23,0); printw("*** [Press any key to quit]");
   refresh();
   getch();
   endwin();
 }

I think if you comment out getch() and maybe refresh() (with a // in front of the line), LIME will quit without asking?

**Comment by lnieblar** [comment]
I tried commenting those out and still couldn't get it to automatically exit when done.

I've decided to just change the `silent` value in `lime.h` to 1, which prevents any output by lime. This seems to work fine.

Also I've updated `nThreads` to 8 and that lets lime compile in less than 2 minutes for me, so that's pretty cool.
